malpolon/models/utils.py

```diff
@@ -118,7 +118,9 @@ def check_loss(loss: nn.modules.loss._Loss) -> nn.modules.loss._Loss:
     """
     if isinstance(loss, nn.modules.loss._Loss):  # pylint: disable=protected-access  # noqa
         return loss
-    raise ValueError(f"Loss must be of type nn.modules.loss. "
+    elif isinstance(loss, str):
+        return eval(loss)
```

Comment:
We are aiming at removing all calls to `eval()` for security purposes. Please take a look at functions `check_optimizer()`, `check_scheduler()` or `check_metrics()`,  on our new branch "no-more-evals" to get an understanding at what a similar `check_loss()` function should look like